CONTRIBUTING.md

```diff
@@ -280,21 +280,30 @@ Linux, MacOS, and cross compiled to Windows using `wine` (on Linux).
 ## Releasing packages to CHaP
 
 New versions of packages are published on [CHaP].  To release packages to
-[CHaP] one should use `./scritp/release-to-chap.sh`.
+[CHaP] one should use `./script/release-to-chap.sh`.
 
 * First run `./script/release-to-chap.sh -r` to see which changes can be
   published.
 * Update versions in `*.cabal` files according to changes in `CHANGELOG.md`
   files.
 * Update `CHANGELOG.md` files.
-* Run `./script/release-to-chap.sh` which will create a PR in
+* Run `./script/release-to-chap.sh` which will create a branch in
   `cardano-haskell-packages` repo (pointed by `CARDANO_HASKELL_PACKAGES_DIR`
   environment variable or `/tmp/chap` if it's not defined).
+  * To enable pushing this branch, cd to the chap repo and execute:
+    `git remote set-url origin git@github.com:IntersectMBO/cardano-haskell-packages.git`
+    then return to the previous repo (`cd -`)
```

Comment:
This would be easily fixed by cloning using the right url, but yeah we can add this here and then try to fix it